### PR TITLE
SS7 upgrade: required amendments to samples

### PR DESCRIPTION
**Pull request by jenmerritt**
### Description

See cs issue 2274.

This PR makes the required changes for an upgrade of SS7. 

- Edit with branch versioning - amended service URL and amended existing credentials
- Configure subnetwork trace - amended service URL and added in credentials
- Display utility associations - amended service URL and added in credentials
- Perform valve isolation trace - amended service URL and added in credentials
- Trace a utility network - amended service URL and added in credentials
- Display subtype feature layer - amended service URL and added in credentials
- Trace a utility network - amended feature table setup and added in credentials

@sclaridge and I have tested the samples (note: have to follow instructions in cs issue 2270 to update hosts file until the upgraded server is live)

@JonLavi @TADraeseke  please could you review so that this PR is ready to be merged whenever they flick the switch on the new server. Thanks!

Checklist:
- [x] Set target branch to master (current release) or dev (next release)
- [x] Request a reviewer
- [x] Assign a reviewer
- [x] Tag reviewer in comment

### CODE REVIEW

File: editing/edit-with-branch-versioning/src/main/java/com/esri/samples/edit_with_branch_versioning/EditWithBranchVersioningController.java [hunk 1]
```diff
@@ -92,10 +92,10 @@ public void initialize() {
 
       // create a service geodatabase
       serviceGeodatabase = new ServiceGeodatabase("https://sampleserver7.arcgisonline" +
-        ".com/arcgis/rest/services/DamageAssessment/FeatureServer");
+        ".com/server/rest/services/DamageAssessment/FeatureServer");
 
       // set the user credentials required to authenticate with the service geodatabase
-      UserCredential userCredential = new UserCredential("editor01", "editor01.password");
+      UserCredential userCredential = new UserCredential("editor01", "S7#i2LWmYH75");
```
Comment (JonLavi) [inline editing/edit-with-branch-versioning/src/main/java/com/esri/samples/edit_with_branch_versioning/EditWithBranchVersioningController.java:105]: This looks like quite a 'serious' password (as opposed to the previous one), are we sure it's supposed to be public? I'm confident this didn't get put out by mistake, but best to make triple sure?

File: feature_layers/display-subtype-feature-layer/src/main/java/com/esri/samples/display_subtype_feature_layer/DisplaySubtypeFeatureLayerController.java [hunk 2]
```diff
@@ -61,18 +63,23 @@ public void initialize() {
...
+
       // set the viewpoint to Naperville, Illinois
       Viewpoint initialViewpoint = new Viewpoint(new Envelope(-9812691.11079696, 5128687.20710657,
         -9812377.9447607, 5128865.36767282, SpatialReferences.getWebMercator()));
       map.setInitialViewpoint(initialViewpoint);
 
       // create a subtype feature layer from the service feature table, and add it to the map
       ServiceFeatureTable serviceFeatureTable = new ServiceFeatureTable("https://sampleserver7.arcgisonline" +
-        ".com/arcgis/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer/100");
+        ".com/server/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer/0");
```
Comment (JonLavi) [inline feature_layers/display-subtype-feature-layer/src/main/java/com/esri/samples/display_subtype_feature_layer/DisplaySubtypeFeatureLayerController.java]: Could we put this on one line instead?

File: feature_layers/display-subtype-feature-layer/src/main/java/com/esri/samples/display_subtype_feature_layer/DisplaySubtypeFeatureLayerController.java [hunk 3]
```diff
@@ -61,18 +63,23 @@ public void initialize() {
...
+
+      // set user credentials to authenticate with the service
+      UserCredential userCredential = new UserCredential("viewer01", "I68VGU^nMurF");
```
Comment (JonLavi) [inline feature_layers/display-subtype-feature-layer/src/main/java/com/esri/samples/display_subtype_feature_layer/DisplaySubtypeFeatureLayerController.java:80]: Same question about this PW - it looks quite realistic, so not sure if it's supposed to be here? If the PW is public, then there's no need in creating a 'secure' password, `viewer01.password` would work just as well and would be memorable if a user tries to play with authentication forms?

There's possibly a few more cases like this in the remainder of the files, so I won't comment on those

**Review by JonLavi — APPROVED** [review]
LGTM @jenmerritt !
I'm sure there's been plenty of thought put into the credentials, but I can't help but question if we need such 'strong' passwords if we put them in public domain? The format `viewer01.password` works well if users want/need to experiment with authentication forms etc, because it's more memorable. If there's another technical reason for the 'strong' passwords, feel free to ignore these comments 😁

File: editing/edit-with-branch-versioning/src/main/java/com/esri/samples/edit_with_branch_versioning/EditWithBranchVersioningController.java (hunk 1 above)
Comment (jenmerritt) [reply editing/edit-with-branch-versioning/src/main/java/com/esri/samples/edit_with_branch_versioning/EditWithBranchVersioningController.java:105]: Thanks for checking! This is the same for all of the platform PRs, these were the requested changes.

Comment (TADraeseke) [reply editing/edit-with-branch-versioning/src/main/java/com/esri/samples/edit_with_branch_versioning/EditWithBranchVersioningController.java:105]: Yep, this caught me for a second too. Yes all these samples should have hardcoded credentials as per the pattern shown here 👍

**Review by TADraeseke — APPROVED** [review]
LGTM! Thanks @jenmerritt

**Comment by jenmerritt** [comment]
Please hold off on merging this PR until the server upgrade has been completed.

**Pull request merged**